tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -230,7 +234,6 @@ class EventFrequencyPercentConditionTestCase(
     SnubaTestCase,
 ):
     rule_cls = EventFrequencyPercentCondition
-    test_event = None
```

Comment:
Having this as a class variable seemed suspect, and makes me wonder if it's carrying over between tests